log.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2022 VMware, Inc. or its affiliates. All Rights Reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package amqp091
+
+import (
+	"log"
+	"os"
+)
+
+var ErrLog = log.New(os.Stderr, "[ERROR] amqp091-go: ", log.LstdFlags|log.Lshortfile)
+var WarnLog = log.New(os.Stderr, "[WARNING] amqp091-go: ", log.LstdFlags|log.Lshortfile)
```

Comment:
One loglevel should be sufficient, all logged messages should only be needed for debugging